src/main/openapi/ds-image-openapi_v1.yaml

```diff
@@ -462,8 +392,150 @@ paths:
               application/xml:
                 schema:
                   $ref: '#/components/schemas/IIIF_info'
+
+  
+  # DeepZoom
+  /deepzoom/{imageid}.dzi:
+    get:
+      tags:
+        - 'Access'
+      summary: 'DeepZoom Image'
+      description: |- 
+        Deep Zoom provides the ability to interactively view high-resolution images. You can zoom in and out of images rapidly without affecting the performance of your application. Deep Zoom enables smooth loading and panning by serving up multi-resolution images and using spring animations.
+      operationId: getDeepzoomImage
+      parameters:
+      - name: imageid
+        in: path
+        description: "Identifier/path for image."
+        required: true
+        schema:
+          type: string
+        example: "image/path.tif"
+      
+      x-streamingOutput: true     
+      responses:
+        '200':
+            description: 'Succes!'
+            content:
+              application/json:
+                schema:
+                  type: object
+                  properties:
+                    TileSize:
+                      type: integer
```

Comment:
Always specify either `int32` or `int64` as `format` when the type is `integer`.